toplevel/coqargs.ml

```diff
@@ -485,6 +491,14 @@ let parse_args ~help ~init arglist : t * string list =
       let opt = to_opt_key opt in
       { oval with config = { oval.config with set_options = (opt, OptionUnset) :: oval.config.set_options }}
 
+    |"-native-output-dir" ->
+      let native_output_dir = next () in
+      { oval with config = { oval.config with native_output_dir } }
+
+    |"-nI" ->
```

Comment:
Wouldn't `-native-include-dir` be more explicit?

Reply:
I dunno, seems a bit excessive to me [taking into account that command-line length is a finite resource] I'm not sure why explicit is important here, as this is really an internal option that users will never see as it will be handled by the build system.